src/couch_jwt_auth.erl

```diff
@@ -58,6 +58,12 @@ decode(Token, Config) ->
       end, List), posix_time(calendar:universal_time()), Config)
   end.
 
+ensure_safe_token(Token, Config) ->
+  case couch_util:get_value(Token, Config) of
```

Comment:
Good stuff, that seems like a cleaner approach. It could also be used across other sections of the code too I guess.
I wonder can I mock `couch_config:get` instead of setting up the whole lot... Would [meck](https://github.com/eproxus/meck) be a good start? (and when I was about to hit Comment I found this https://github.com/apache/couchdb-meck :P)